apps/dashboard/app/views/projects/show.html.erb

```diff
@@ -31,7 +31,7 @@
 </div>
 
 <div class="row mb-3">
-  <div class="col-md-2">
+  <div class="col-md-3">
```

Comment:
Can we keep this at 2? I feel like we need the real estate for other things to come like file navigation and job inspection.